frontend/src/metabase/query_builder/components/expressions/ExpressionEditorTextfield/ExpressionEditorTextfield.tsx

```diff
@@ -446,6 +447,11 @@ class ExpressionEditorTextfield extends React.Component<
   };
 
   handleInputBlur = (e: React.FocusEvent) => {
+    // Ensure there is no active popup menu before we blur
+    if (this.popupMenuTarget?.current) {
+      return;
+    }
```

Comment:
This change causes `ExpressionWidget.unit.spec.tsx` and `FilterPicker.unit.spec.tsx` tests to fail

Reply:
@kamilmielnik apologies, this is the fix - https://github.com/metabase/metabase/compare/master...hyperNURb:ivan/41165_scrollbar_suggestion_interaction?expand=1

Reply:
Should I open a followup PR?

Reply:
> Should I open a followup PR?

Please do, we are [reverting](https://github.com/metabase/metabase/pull/43992) the original one because it prevents us from merging things to `master`.